src/robotlibcore.py

```diff
@@ -141,6 +131,29 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, name):
+        kw = self.keywords[name]
+        if hasattr(kw, 'robot_types'):
+            robot_types = getattr(kw, 'robot_types')
+            if robot_types:
+                return robot_types
```

Comment:
This isn't right. Setting `robot_types` to `None` (i.e. `@keyword(types=None)`) should disable type conversion altogether. See the related code on Robot side here:
https://github.com/robotframework/robotframework/blob/master/src/robot/running/arguments/argumentparser.py#L78